xAPI.md

```diff
@@ -306,10 +306,21 @@ be unintuitive and/or lengthy to dissect into a list of requirements.
 
 ## 4.0 Binding to JavaScript Object Notation (JSON)
 
+This specification makes a specific binding to JSON for most of the data structures allowed.  There are some 
```

Comment:
Also, JSON is only recommended for document types in certain cases. 

Perhaps this section need to be a lot shorter to avoid needing to explain in detail. "The spec uses JSON, there could be other data bindings, there aren't any that actually exist."